components/data_comps/desp/esp_utils.F90

```diff
@@ -0,0 +1,344 @@
+module esp_utils
+
+! !USES:
+
+  use shr_kind_mod, only: r8=>shr_kind_r8, CL=>SHR_KIND_CL
+  use shr_sys_mod,  only: shr_sys_abort, shr_sys_flush
+
+  implicit none
+  private
+#include <mpif.h>
+  save
```

Comment:
I don't think we want blanket save statements in modules and shouldn't we use mpi instead of include <mpif.h>?